ts/client/src/client.ts

```diff
@@ -1449,6 +1520,86 @@ export class MangoClient {
     );
   }
 
+  public async serum3LiqForceCancelOrders(
+    group: Group,
+    mangoAccount: MangoAccount,
+    externalMarketPk: PublicKey,
+    limit?: number,
+  ): Promise<string> {
+    const serum3Market = group.serum3MarketsMapByExternal.get(
+      externalMarketPk.toBase58(),
+    )!;
+    const serum3MarketExternal = group.serum3ExternalMarketsMap.get(
+      externalMarketPk.toBase58(),
+    )!;
+
+    const healthRemainingAccounts: PublicKey[] =
+      this.buildHealthRemainingAccounts(
+        AccountRetriever.Fixed,
+        group,
+        [mangoAccount],
+        [
+          group.getFirstBankByTokenIndex(serum3Market.baseTokenIndex),
+          group.getFirstBankByTokenIndex(serum3Market.quoteTokenIndex),
```

Comment:
unnecessary since base and quote positions are guaranteed to exist on the account